core/cli/commands/extract.py

```
@@ -0,0 +1,101 @@
+            equations=len(result.get("equations", [])),
+            images=len(result.get("images", [])),
+        )
+
+    except ImportError as e:
+        return error_response(
+            command="extract",
+            code=ErrorCode.CONFIG_ERROR,
+            message=f"Extraction dependencies not available: {e}",
+            start_time=start_time,
+        )
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)